src/rviz/ogre_helpers/render_system.cpp

```diff
@@ -184,10 +183,6 @@ void RenderSystem::detectGlVersion()
     int major = caps->getDriverVersion().major;
     int minor = caps->getDriverVersion().minor;
     gl_version_ = major * 100 + minor * 10;
-
-    std::string gl_version_string = (const char*)glGetString(GL_VERSION);
-    // The "Mesa 2" string is intended to match "Mesa 20.", "Mesa 21." and so on
-    mesa_workaround = gl_version_string.find("Mesa 2") != std::string::npos && gl_version_ >= 320;
```

Comment:
My suggestion would have been to just change this to "Mesa 20"

Reply:
Was the Mesa 20 part of some HWE or part of the stock 18.04, ie are there likely going to be people with non HWE systems running into this again?

Reply:
```
$ LANG=C apt-cache policy mesa-va-drivers 
mesa-va-drivers:
  Installed: 21.3.6~kisak1~b
  Candidate: 21.3.6~kisak1~b
  Version table:
 *** 21.3.6~kisak1~b 500
        500 http://ppa.launchpad.net/kisak/kisak-mesa/ubuntu bionic/main amd64 Packages
        100 /var/lib/dpkg/status
     20.0.8-0ubuntu1~18.04.1 500
        500 http://cz.archive.ubuntu.com/ubuntu bionic-updates/universe amd64 Packages
     19.2.8-0ubuntu0~18.04.2 500
        500 http://security.ubuntu.com/ubuntu bionic-security/universe amd64 Packages
     18.0.0~rc5-1ubuntu1 500
        500 http://cz.archive.ubuntu.com/ubuntu bionic/universe amd64 Packages
```

Reply:
That's on Bionic HWE with kisak-mesa PPA. So it seems 20.x is available via bionic-updates for everyone.